.github/workflows/test-file-tester.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-20.04
     env:
         PACKAGE_NAME: file-tester
-        SRC_DIR: utils/file-tester/extractor-lib
+        SRC_DIR: utils/file-tester/lms_file_tester
```

Comment:
Somehow we never noticed this error before. Although not related to the ticket at hand, I didn't want to continue moving forward with a breaking build process. Now that this build job is fixed, you'll see that I also had to make some type adjustments in the file-tester files.